packages/account/src/utils/formatTransferToContractScriptData.ts

```diff
@@ -4,7 +4,35 @@ import { BN, type BigNumberish } from '@fuel-ts/math';
 import * as asm from '@fuels/vm-asm';
 import { getBytesCopy, type BytesLike } from 'ethers';
 
-export const composeScriptForTransferringToContract = async () => {
+interface IAssembleTransferToContractScriptParams {
+  hexelifiedContractId: B256Address;
```

Comment:
This should be `hexlifiedContractId` but I can't make a typo fix suggestion as it spans multiple files.